app/lib/frontend/templates/publisher.dart

```diff
@@ -132,3 +161,22 @@ String renderPublisherAdminPage(Publisher publisher) {
     noIndex: true,
   );
 }
+
+Tab _packagesLinkTab(String publisherId) => Tab.withLink(
+      id: 'packages',
+      title: 'Top Packages',
+      href: urls.publisherUrl(publisherId),
+    );
+
+Tab _aboutLinkTab(String publisherId) => Tab.withLink(
+      id: 'about',
```

Comment:
I'm not sure we need this about page... but maybe that's something we discuss with mit when we get further..